fuzzy_test.go

```diff
@@ -345,7 +345,7 @@ func TestAutocomplete(t *testing.T) {
 	model.Train(sampleEnglish)
 	out, err := model.Autocomplete("accoun")
 	if err != nil {
-		t.Errorf("Auocomplete() returned and error: ", err)
+		t.Errorf("Auocomplete() returned an error: %s", err)
```

Comment:
While we are here can we fix the spelling of "autocomplete"?

Reply:
Yep!  Feels like I would have spotted that the first time around.  I'll make those two updates.

Reply:
"Auocomplete" -> "Autocomplete" :)